src/server/logging/log_interceptor.js

```diff
@@ -70,13 +94,23 @@ export class LogInterceptor extends Stream.Transform {
    *  @param {object} - log event
    */
   downgradeIfEcanceled(event) {
-    return downgradeIfErrorMatches('ECANCELED', event);
+    return downgradeIfErrorType('ECANCELED', event);
+  }
+
+  downgradeIfHTTPSWhenHTTP(event) {
+    return downgradeIfErrorMessage('Parse Error', event);
```

Comment:
Is error message matching our only option here? `Parse Error` is pretty vague isn't it? I guess we're still limiting it by the tag list, but I'd really like to use a value that's intended for machines (like `errno`) rather than one that's intended for humans (and much more likely to change/conflict)